interface/modules/custom_modules/oe-module-claimrev-connect/public/claims.php

```diff
@@ -0,0 +1,334 @@
+<?php   
+    namespace OpenEMR\Modules\ClaimRevConnector;
+    $tab="claims";
+    require_once "../../../../globals.php";
+   
+    require_once '../src/ClaimSearchModel.php';
+    require_once '../src/ClaimSearch.php';
+    require_once '../src/ClaimRevApi.php';
+    require_once '../src/UploadEdiFileContentModel.php';
+
+   
+    use OpenEMR\Modules\ClaimRevConnector\ClaimSearch;
+    use OpenEMR\Modules\ClaimRevConnector\ClaimSearchModel;
+
+    class ClaimsPage
+    {
+        public static function SearchClaims($postData)
+        {
+            $firstName = $_POST['patFirstName']; 
+            $lastName = $_POST['patLastName']; 
+            $startDate = $_POST['startDate']; 
+            $endDate = $_POST['endDate']; 
+
+            $model = new ClaimSearchModel();
+            $model->patientFirstName = $firstName;
+            $model->patientLastName = $lastName;
+            $model->receivedDateStart = $startDate;
+            $model->receivedDateEnd = $endDate;
+
+            $data = ClaimSearch::Search($model);
+            return $data;
+        }
+    }
+?>
+<html>
+    <head>
+        <link rel="stylesheet" href="../../../../../public/assets/bootstrap/dist/css/bootstrap.min.css">
+    </head>
+    <title>ClaimRev Connect - Claims</title>
+    <body>
+        <div class="row"> 
+            <div class="col">
+            <?php
+               
+                include '../templates/navbar.php';
+            ?>
+            </div>
+        </div>
+        <form method="post" action="<?=$_SERVER['PHP_SELF'];?>">
+            <div class="card">  
+                <div class="row">
+                    <div class="col">
+                        <div class="form-group">
+                            <label for="startDate">Send Date Start</label>
+                            <input type="date" class="form-control"  id="startDate" name="startDate" value="<?php echo isset($_POST['startDate']) ? $_POST['startDate'] : '' ?>"  placeholder="yyyy-mm-dd"/>
```

Comment:
Need to html escape above `$_POST['startDate']`
When html escaping, there are 2 main functions. attr() when escaping within an html attribute and text() when escaping outside an html attribute. So, in this case would use attr():
```
<input type="date" class="form-control"  id="startDate" name="startDate" value="<?php echo isset($_POST['startDate']) ? attr($_POST['startDate']) : '' ?>"  placeholder="yyyy-mm-dd"/>
```